docs/Features/RelaxNGFeatures.md

```diff
@@ -0,0 +1,135 @@
+# RelaxNG features
+
+[vscode-xml](https://github.com/redhat-developer/vscode-xml) provides validation, completion, hover [RelaxNG](https://relaxng.org/) support in XML file since `0.22.0` (prerelease):
+
+![RelaxNG support](../images/RelaxNG/RelaxNGSupport.gif)
+
+It supports both:
+
+ * [RelaxNG XML syntax (*.rng files)](https://relaxng.org/tutorial-20011203.html).
+ * [RelaxNG Compact syntax (*.rnc files)](https://relaxng.org/compact-tutorial-20030326.html).
+
+![XML Validated with RelaxNG](../images/RelaxNG/XMLValidatedWithRelaxNG.png)
+
+You can associate your XML file with `RelaxNG grammar` (`file` or `URL`) using :
+
+ * an `xml-model` processing instruction
+
+```xml
+<?xml-model href="addressBook.rng" type="application/xml"?>
+<addressBook>   
+  ...
+</addressBook> 
+```
+
+ * an `XML catalog` like you can do that [with XSD](Validation#XML-catalog-with-XSD)
+ * a `file association` like you can do that [with XSD](Validation#XML-file-association-with-xsd)
+
+## RelaxNG grammar
+
+[vscode-xml](https://github.com/redhat-developer/vscode-xml) provides a support for `RelaxNG grammar`:
+
+ * for [RelaxNG XML syntax (*.rng files)](https://relaxng.org/tutorial-20011203.html), it provides the same support as XSD/DTD grammars (completion, hover, validation, etc).
+ 
+![RelaxNG XML Syntax Overview](../images/RelaxNG/RelaxNGXMLSyntaxOverview.png)
+  
+ * for [RelaxNG Compact syntax (*.rnc files)](https://relaxng.org/compact-tutorial-20030326.html) it provides only syntax coloration.
+ 
+![RelaxNG Compact Syntax Overview](../images/RelaxNG/RelaxNGCompactSyntaxOverview.png)
+
+# Validation
+
+XML validation based on RelaxNG (rng, rnc) is supported:
+
+![RelaxNG Validation](../images/RelaxNG/RelaxNGValidation.png)
+
+# Completion
+
+XML completion based on RelaxNG (rng, rnc) is supported. The completion for rng can show the documentation:
+
+![RelaxNG Completion And Documentation](../images/RelaxNG/RelaxNGCompletionAndDocumentation.png)
+
+# Hover
+
+Hover based on RelaxNG rng can show the documentation:
+
+![RelaxNG Hover And Documentation](../images/RelaxNG/RelaxNGHoverAndDocumentation.png)
+
+# Go to Type Definition
+
+From the XML you can go to the type definition to navigate to the element/attribute declaration for rnc, rng both.
+
+To do that, select an XML element/attribute and use the contextual meny `Go to Type Definition`: 
+
+![Go to Type Definition Menu](../images/RelaxNG/GoToTypeDefinitionMenu.png)
+
+The click on this menu item, will open the rnc, rng grammar file and set the cursor to the proper element/attribute declaration:

```

Comment:
*When you* click on this menu item, *VS Code* will open the rng *or* rng grammar file and *place* the cursor *on* the proper element/attribute declaration:

Reply:
fixed